hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -348,6 +353,12 @@ If no USB mass storage devices are found within this timeout then USB-MSD is sto
 Default: 20000 (20 seconds)  
 Version: pieeprom-2020-05-15.bin - BETA  
 
+### USB_MSD_BOOT_MAX_RETRIES
+The number of times USB MSD boot will be retried after failure before moving to the next boot mode defined by `BOOT_ORDER`.
```

Comment:
Does this have the same "-1 means infinite retries" behaviour as the other `_MAX_RETRIES` options?

For consistency, I wonder if the "before moving to the next boot mode defined by `BOOT_ORDER`" text used here should also be added to the descriptions of the other `_MAX_RETRIES` options?

Reply:
ok